src/panels/lovelace/common/create-card-element.js

```diff
@@ -19,7 +19,7 @@ import "../cards/hui-picture-elements-card";
 import "../cards/hui-picture-entity-card";
 import "../cards/hui-picture-glance-card";
 import "../cards/hui-plant-status-card";
-import "../cards/hui-sensor-card";
+import "../cards/hui-sensor-card.ts";
```

Comment:
Should be no need to include the extension here

Reply:
I thought that too. Ill try it again but it was broken for me